silverpeas/content.md

```diff
@@ -0,0 +1,180 @@
+-	the Microsoft SQLServer database system,
+-	the Oracle database system.
+
+The Silverpeas images support actually only the two first database systems; because of the non-free licensing issues with the Oracle JDBC drivers, Silverpeas cannot include these drivers by default and consequently it cannot use transparently an Oracle database system as a persistence backend.
+
+For the same reasons, the Docker images of Silverpeas aren't shipped with the Oracle JVM but with the default OpenJDK. Silverpeas uses the Wildfly application server as runtime.
+
+The Silverpeas images use the following environment variables to set the database access parameters:
+
+-	`DB_SERVERTYPE` to specify the database system to use with Silverpeas: POSTGRESQL for PostgreSQL, MSSQL for Microsoft SQLServer, ORACLE for Oracle. By default, it is set to POSTGRESQL.
+-	`DB_SERVER` to specify the IP address or the name of the host on which the database system is running. By default, it is set to `database`, so that any container running a database can be linked to the Silverpeas container with this name.
+-	`DB_NAME` to specify the database to use with Silverpeas. By default, it is set to `Silverpeas`.
+-	`DB_USER` to specify the user identifier to use by Silverpeas to access the database. By default, it is set to `silverpeas` (it is recommended to dedicate a user account in the database for each application).
+-	`DB_PASSWORD` to specify the password associated with the user identifier above.
+
+These environment variables can be also defined as properties into the Silverpeas global configuration file `config.properties` (see below).
+
+## Start a Silverpeas instance with a database from a Docker container
+
+In [Docker Hub](https://hub.docker.com/), no Docker images of Microsoft SQLServer are currently available, but you will find a lot of images of PostgreSQL. For example, with an [official PostgreSQL docker image](https://hub.docker.com/_/postgres/), you can start a PostgreSQL instance initialized with a superuser `postgres` with as password `mysecretpassword`:
+
+	$ docker run --name postgresql -d \
+	    -e POSTGRES_PASSWORD="mysecretpassword" \
+	    -v postgresql-data:/var/lib/postgresql/data \
+	    postgres:9.6
+
+We recommend strongly to mount the directory with the database file on the host so the data won't be lost when upgrading PostgreSQL to a newer version (a Data Volume Container can be used instead). For any information how to start a PostgreSQL container, you can refer its [documentation]((https://hub.docker.com/_/postgres/).
+
+Once the database system is running, a database for Silverpeas has to be created and a user with administrative rights on this database should be added; it is recommended to create a dedicated user account in the database for each application and therefore for Silverpeas. In this document, and by default, a database `Silverpeas` and a user `silverpeas` for that database are created.
```

Comment:
Wouldn't it be easier for users to just use `-e POSTGRES_USER=silverpeas` and `-e POSTGRES_DB=Silverpeas` on the postres container creation?  When running a postgres container, most users create a new container for each application that needs a database, rather than having one database container for all of them.